lib/message-queue.js

```diff
@@ -1,14 +1,13 @@
 var _ = require('lodash');
 
-// TODO... what indexes should be created and should this be responsible for creating them?
-
 function MessageQueue() {
   var self = this;
 
   self.errorHandler = console.error;
 
   self.databasePromise = null;
   self.collectionName = '_queue';
+  self.createIndex = false;
```

Comment:
We may want to default this to true

Reply:
I was considering defaulting to `true` being unexpected for a minor version bump. If I were an external consumer of this API, I might not appreciate having an index created for me automatically by upgrading a minor version. It's not "technically" an API break, but it's unexpected.  We could make a major version bump.  We're also forcing a minimum engine with this change as well... Thoughts?

Reply:
Yeah, I was thinking we would do a major version bump.  Probably 1.0.0 finally.  We are forcing node 8+, es6, and native async await all of which feel more breaking than adding a new index feature.